parlai/mturk/tasks/acute_eval/README.md

```diff
@@ -1,253 +1,5 @@
-
-    python parlai/mturk/tasks/acute_eval/fast_eval.py --id-pairs <comma-separated, colon-delimited list of config identifiers>
-
-The ids specified for each of those flags corresponds to the entry in the `CONFIG`.
-
-If you are running self-chat, you can optionally specify a seed task to use for self-chat with `-t <self_chat_task>`.
-
-A few examples are as follows:
-
-    python parlai/mturk/tasks/acute_eval/fast_eval.py --ids  example_model_1,example_model_2,example_model_log,example_dataset -t blended_skill_talk
-
-    python parlai/mturk/tasks/acute_eval/fast_eval.py --id-pairs  example_model_1:example_model_2,example_model_1:example_model_log,example_dataset:example_model_2 -t blended_skill_talk
-
-When you are ready to run a **LIVE** ACUTE-Eval, please specify `--live-acute true`.
-
-#### Onboarding
-
-The default onboaring dialogue pair is in `example/onboarding.json`. We recommend you use a different onboarding example as the one provided is quite easy.
-
-To use a custom onboarding path, specify the `--onboarding-path` when running `fast_eval.py`. The onboarding file should be a jsonl file, where each line is a json dict consisting of a pair of dialogues to evaluate, and where `is_onboarding` is set to True.
-
-## Script Execution
-
-The script operates in three phases:
-
-#### Phase 1: Compile Chat Logs
-
-The script will first compile the chat logs for each identifier specified on the command line.
-
-For `model`s, the script will run self-chat (if a self-chat log does not already exist); for `log`s, the script will simply load the log from disk; and for `task`s, the script will convert the task into the appropriate format.
-
-Self-chats are saved to `PARLAI_PATH/data/acute_evals/self_chats/`
-
-### Phase 2: ACUTE-Eval
-
-The script will then prepare the conversation-pairs file (and save to `PARLAI_PATH/data/pairings_files/`, unique according to which chat files were used to create it) and run ACUTE-Eval with appropriate arguments.
-
-Upon subsequent runs with the same configuration of `--ids` or `--id-pairs`, you will have the option to re-use a pairings file or to regenerate it.
-
-### Phase 3: Analysis
-
-After finishing ACUTE-Eval, the script will analyze and save relevat results to `PARLAI_PATH/data/acute_evals/acute_results/<date>/<pairings_file>/`
-
-4 Results will be generated:
-
-1. A csv file of significance result, which shows the win rates of model pairs with p value
-2. A csv file of grid result, where the model comparisons and win rates are laid out in a nice grid (as seen in the ACUTE-Eval paper).
-3. A html file of nicely visualized conversations with reason annotated only
-4. A html file of ALL nicely visualized conversations
-
-**NOTE** the the `analysis.py` file can be run by itself as long as you specify the ACUTE-Eval `run_id`, whether a sandbox run, and whether it is qfunction eval.
+This task has been archived. You can restore it by switching to the `acute_eval` tag of ParlAI:
```

Comment:
Nit - I feel this note should first direct people to use the Mephisto version.

Reply:
Definitely - changing